server/sql.go

```diff
@@ -752,8 +752,8 @@ func (s *Server) sqlCheck(ctx context.Context, instance *api.Instance, exec *api
 			adviceList = append(adviceList, advisor.Advice{
 				Status:  advisor.Warn,
 				Code:    common.TaskCheckEmptySchemaReviewPolicy,
-				Title:   "Empty schema review policy or disabled",
-				Content: "",
+				Title:   "",
```

Comment:
Could you tell me the reason for this modification?

Reply:
For `Issue`, we show this advice as
![0w1EduGkzb](https://user-images.githubusercontent.com/20775801/174569361-f0a4fdab-0a6a-4696-a51e-faf63ec5c92b.png)
By design, this message is indeed the title. I would like to be consistent with them. WDYT.